### PR TITLE
Add libraw

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/libraw/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+autoreconf --install
+./configure --prefix="${PREFIX}"
+
+make -j${CPU_COUNT}
```
Comment (scopatz) [inline recipes/libraw/build.sh]: Is there a `make check` you can run?

File: recipes/libraw/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - automake
+    - libtool
+    - pkg-config
+    - {{ compiler('cxx') }}
```
Comment (scopatz) [inline recipes/libraw/meta.yaml]: make should also be in this list

File: recipes/libraw/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - jpeg
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  run:
+    - {{ pin_compatible('lcms2', max_pin='x.x') }}
+
+about:
+  home: http://www.libraw.org/
+  license: LGPL3
+  license_file: LICENSE.LGPL
```
Comment (scopatz) [inline recipes/libraw/meta.yaml]: You should have `license_family` as an entry here too

File: recipes/libraw/build.sh (hunk 1 above)
Comment (wolfv) [reply recipes/libraw/build.sh]: I didn't find one :/

File: recipes/libraw/meta.yaml (hunk 2 above)
Comment (wolfv) [reply recipes/libraw/meta.yaml]: :+1:

File: recipes/libraw/meta.yaml (hunk 3 above)
Comment (wolfv) [reply recipes/libraw/meta.yaml]: :+1:

**Pull request merged**